tests/Unit/Schema/Directives/UploadDirectiveTest.php

```diff
@@ -0,0 +1,218 @@
+<?php
+
+namespace Tests\Unit\Schema\Directives;
+
+use Exception;
+use Illuminate\Http\UploadedFile;
+use Illuminate\Support\Facades\Storage;
+use Tests\TestCase;
+
+final class UploadDirectiveTest extends TestCase
+{
+    private const DISK_NAME = 'uploadDisk';
+
+    protected $operations = [
```

Comment:
Use a local variable instead, I would rather duplicate some test code but have it more readable. DRY is not a big concern in testing.